### DISCUSSION TITLE
Media Encoding is Pending ... And Stuck

### DISCUSSION THREAD

**Discussion by hquimby** [mediacms-io/mediacms#1069] [Q&A]
UPDATE:  I haven't been able to reproduce this.  Please delete this thread.

I have been loving MediaCMS running well in a Promox VM under Ubuntu 22.04.  I followed the single server installation steps and upgrade.  Media encoding and content management were fine until today's upload.  Today's file is smaller than files I've done before.  I have deleted and re-tried the upload.  I renamed and re-tried it.  I have rebooted and then completely restarted the VM.  Reading other threads, I have tried the trick of switching media type from video to ------ and then back, but that doesn't work either.  Encoding just won't start.

The only thing I found of interest in the logs is the following from errorlog.txt:

/home/mediacms.io/lib/python3.10/site-packages/rest_framework/pagination.py:200: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'files.models.Playlist'> QuerySet.
  paginator = self.django_paginator_class(queryset, page_size)

Any ideas on how to fix this?  TIA.

And if it requires a re-install, is there a way to backup the current content and import it back in so I don't lose everything?

**Comment by mgogoulos** [comment]
Hi, wondering if the problem is related to a specific file or not. 
Can you try with a file that is already uploaded? Is it getting uploaded? Perhaps the file is not recognized as video and the encoding does not start? 

Also did you update the codebase, or have been running without changes?

**Reply by hquimby** [reply]
I think I caused the issue when I turned off 2 encoding profiles (240 and 360) and deleted the associated low res encodings.  I restored from an earlier backup and things are working as expected again.

I'll try those deletes again in a different VM and see if I can re-create how I broke it.

**Comment by hquimby** [comment]
UPDATE:  I haven't been able to reproduce this and it's been running fine for over a month.  Please delete this thread.

**Comment by mgogoulos** [comment]
I guess the problem has been that Encoding Profiles 240 and 360 were disabled! Glad you resolved this
